integration/helpers.go

```diff
@@ -193,6 +193,19 @@ func WithRuntimeMetadata(runtime string) CommonDataOption {
 	}
 }
 
+// WithCommonDataContainerImage sets the ContainerImageName to facilitate the tests
+func WithCommonDataContainerImage(containerImage string, isDockerRuntime bool) CommonDataOption {
+	return func(commonData *eventtypes.CommonData) {
+		commonData.Runtime.ContainerImageName = containerImage
+		// TODO: Handle it once we support getting container image name for docker.
+		// As docker do not provide OCI annotations, we need to find another way
+		// to retrieve image name for new containers.
+		if isDockerRuntime {
+			commonData.Runtime.ContainerImageName = ""
+		}
```

Comment:
```suggestion
		// TODO: Handle it once we support getting container image name for docker.
		// As docker do not provide OCI annotations, we need to find another way
		// to retrieve image name for new containers.
		if !isDockerRuntime {
			commonData.Runtime.ContainerImageName = containerImage
		}
```
?

Reply:
Good idea! Thank you. Added in a separate commit.